discussment-core/src/main/java/org/danekja/discussment/core/accesscontrol/service/DiscussionUserService.java

```diff
@@ -36,4 +39,12 @@ default boolean isGuest() {
         return getCurrentlyLoggedUser() == null;
     }
 
+    /**
+     * Returns a list of users with given ids.
+     * @param userIds
+     * @return
+     */
+    default List<IDiscussionUser> getUsersByIds(Collection<String> userIds) {
+        return Collections.emptyList();
```

Comment:
This default implementation is completely useless. For UI to work correctly, you _have to_ implement this method - so it's BC break no matter if the default implementation is here or not.

Drop `default` so that compiler forces devs to implement the method appropriately. Returning empty list is never correct.